packages/terra-section-header/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-section-header",
+  "main": "lib/SectionHeader.js",
+  "version": "0.0.0",
+  "description": "Interactive header component that provides a status indicator.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-section-header",
+    "SectionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
```

Comment:
terra-props-table can be removed. This was updated in #1301

Reply:
Done df26fdb35fbf4ea05832b3c24f9d3d132f1bffa1